concepts/05 Widgets/Diagram/10 Data Binding/40 Data Binding Options.md

```diff
@@ -0,0 +1,330 @@
+The **Diagram** widget allows you to bind [node's](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes) and [edge's](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/edges) settings to a database. Binding options have postfix _'Expr'_ and can contain a name of a data source field or an expression that returns the corresponding node's or edge's data.
+
+    <!--JavaScript-->
+    $(function() {
+        $("#diagram").dxDiagram({
+            nodes: {
+                dataSource: new DevExpress.data.ArrayStore({
+                    key: "this",
+                    data: orgItems
+                }),
+                keyExpr: "ID",
+                textExpr: "Title",
+                parentKeyExpr: "Head_ID",
+                typeExpr: itemTypeExpr,
+                styleExpr: itemStyleExpr,
+            },
+        });
+        function itemTypeExpr(obj, value) {
+          return obj.type === "group" ? "ellipse" : "rectangle";
+        }
+        function itemStyleExpr(obj) {
+            return { "stroke": "red" };
+        }
+    });
+   
+        
+![Diagram - Data Binding Options](/images/diagram/binding-options.png)
+
+When you bind **Diagram** to a data source you must specify options that allow the widget to create a diagram structure. A set of required options depends on the source data structure.
```

Comment:
Can we say:
When you bind **Diagram** to a data source, specify widget options to allow diagram structure creation. Required options depend on the source data structure.